### ISSUE TITLE
Wallaby.assert_no_violations/2 does render the output of liveview slots

### ISSUE THREAD

**Issue by badeyeye1** [angelikatyborska/a11y-audit-elixir#36]
Accessibility violation detected when there is none.

Using `Wallaby.assert_has(Query.css("h1", text: "Log in"))` shows that the heading contains a text.

```elixir
     Expected page to have no accessibility violations, but got 1 violation.

      minor  Headings should not be empty
     ┃ Learn more: https://dequeuniversity.com/rules/axe/4.10/empty-heading?application=axeAPI
     ┃
     ┃ There is 1 node with this violation:
     ┃
     ┃ 1. <h1 class="text-lg font-semibold leading-8 text-white">\n      \n    \n    </h1>
     ┃
     ┃    Fix any of the following:
     ┃      - Element does not have text that is visible to screen readers
     ┃      - aria-label attribute does not exist or is empty
     ┃      - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
     ┃      - Element has no title attribute
```

**Comment by angelikatyborska** [comment]
Hi! Thank you for reporting, but this is not enough information for me to try to help. Could you share more of your code? How does your test code exactly look like? How does your page template and components look like? Are you able to recreate this problem in a small test project so I could actually run the code myself?

Some general advice: just because you have a wallaby assertion that there is an `h1` on your page that has the text "Log in" does not mean that there isn't also an empty `h1`. You can potentially have multiple `h1`s on your page and one of them is empty. The violation report gives you this markup: `<h1 class="text-lg font-semibold leading-8 text-white">\n      \n    \n    </h1>` Make sure to do a global search in your project for those classes to find the exact `h1` element that causes the violation. Maybe it's a different h1 than you expected?

Another problem could be the timing - is it possible that you're first rendering your page with some content missing and only later filling out that content. E.g. in live view, the page renders before the socket connects and then after. Maybe you don't have any data before the socket connects?

**Comment by badeyeye1** [comment]
I would recreate in a small project and share here

**Comment by angelikatyborska** [comment]
Did you manage to recreate the bug in a new project that you could share with me?

**Comment by badeyeye1** [comment]
After updating my chromedriver, I could not reproduce it. Closing this now. Thanks
